modules/kafka/src/main/java/org/testcontainers/containers/KafkaContainer.java

```diff
@@ -136,6 +144,9 @@ protected void configure() {
     }
 
     protected void configureKraft() {
+        //CP 7.4.0
+        withEnv("KAFKA_CONTROLLER_QUORUM_MODE", "kraft");
```

Comment:
We changed this logic so this env var is no longer necessary.  As long as you are using KAFKA_PROCESS_ROLES (process.roles) it means you are in KRaft mode.  See https://github.com/confluentinc/cp-all-in-one/blob/7.4.0-post/cp-all-in-one-kraft/docker-compose.yml#L22